browser-test/src/support/index.ts

```diff
@@ -439,6 +438,15 @@ export const validateScreenshot = async (
   if (DISABLE_SCREENSHOTS) {
     return
   }
+  // To make screenshots stable go through all date fields (elements that have cf-bt-date class) and replace date/time with fixed text.
+  await element.evaluate(() => {
+    for (const date of Array.from(document.querySelectorAll('.cf-bt-date'))) {
+      console.log('found date ' + date.textContent)
+      date.textContent = date
+        .textContent!.replace(/\d{4}\/\d{2}\/\d{2}/, '2030/01/01')
```

Comment:
What about putting only the date component in a `<span>` element and just replacing the text content with 2030/01/01?

Then if we wanted date time, we could add a cf-bt-datetime class and add more to it.

Reply:
Having regexp replace provides benefit that if underlying format changes the regex either will still work, replacing part of the text, or will break, causing screenshot change. Fully replacing text content has potential issue of hiding underlying changes in rendering. For example we decided to render full date like "September 20, 2022".

Reply:
Works for me! I'd consider adding something along these lines as a comment in the code for future readers.